static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+              "description": "The count of unique unoptimized pages."
+            }
+          },
+          "required": [
+            "optimizedReqCount",
+            "passthroughReqCount",
+            "unoptimizedReqCount",
+            "optimizedPageCount",
+            "passthroughPageCount",
+            "unoptimizedPageCount"
+          ]
+        },
+        "MetricsSummaryResponse": {
+          "allOf": [
+            {
+              "$ref": "#/components/schemas/MetricsBaseResponse"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "analysisTimeSeconds": {
+                  "type": "number",
+                  "description": "The average number of seconds required to run analysis."
+                },
+                "validationTimeSeconds": {
+                  "type": "number",
+                  "description": "The average number of seconds required to run validation."
+                }
+              }
+            }
+          ]
+        },
+        "MetricsDetailsResponse": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/MetricsBaseResponse"
+              },
+              {
+                "type": "object",
+                "properties": {
+                  "dateGroup": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The timestamp group"
+                  }
+                }
+              }
+            ]
+          }
+        },
+        "MetricsResponse": {
+          "type": "object",
+          "properties": {
+            "summary": {
+              "$ref": "#/components/schemas/MetricsSummaryResponse"
+            },
+            "details": {
+              "$ref": "#/components/schemas/MetricsDetailsResponse"
+            }
+          }
+        }
+      }
+    },
+    "paths": {
+      "/api/p3/v1/version": {
+        "get": {
+          "summary": "Version of service",
+          "tags": [
+            "Version"
+          ],
+          "security": [
+            {
+              "ApiKeyAuth": []
+            }
+          ],
+          "description": "Get current version of the service.",
+          "operationId": "getVersion",
+          "responses": {
+            "200": {
+              "description": "Successfully fetched version.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/GetVersionResponse"
+                  }
+                }
+              }
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/health": {
+        "get": {
+          "summary": "Service health status",
+          "tags": [
+            "Health"
+          ],
+          "security": [
```

Comment:
Be aware most of our services do not have auth protection against health.  It makes services like route53 more complex to check health if you have to also provide an apikey.